### PR TITLE
Make scoring part more general

### PR DESCRIPTION
**Pull request by siddhu001**

### CODE REVIEW

**Comment by siddhu001** [comment]
@YushiUeda @pyf98 @yuekaizhang @SujaySKumar can you please look at this pull request and standardise the scoring function in your recipe to use the egs2/TEMPLATE/asr1/pyscripts/utils/score.py after this pull request has merged. Thanks!

File: egs2/TEMPLATE/asr1/pyscripts/utils/score_intent.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        hyp_intent = hyp_lines[line_count].split(" ")[0]
+        ref_intent = ref_lines[line_count].split(" ")[0]
+        if hyp_intent != ref_intent:
+            error += 1
+    return 1 - (error / len(hyp_lines))
+
+
+exp_root = sys.argv[1]
+
+if len(sys.argv) == 4:
```
[the remark this answers is not in the document]
Comment (siddhu001) [reply egs2/TEMPLATE/asr1/pyscripts/utils/score_intent.py]: Done!

**Comment by sw005320** [comment]
Thanks, @siddhu001!
It looks great!

One last request.
Can you also add a test to https://github.com/espnet/espnet/tree/master/test_utils?
This will make your script safe.
The scoring part should not have any mistakes.

**Comment by siddhu001** [comment]
Sure I will add it!

**Pull request merged**